code/client/src/sdk/entities/c_car.cpp

```diff
@@ -91,8 +97,8 @@ namespace SDK {
         hook::this_call<void>(gPatterns.C_Car__SetSpeed, this, speed);
     }
 
-    void C_Car::PosefujZimuVShopu(float dirt) {
-        hook::this_call<void>(gPatterns.C_Car__PosefujZimuVShopu, this, dirt);
+    void C_Car::SetVehicleDirty(float dirt) {
+        hook::this_call<void>(gPatterns.C_Car__SetVehicleDirty, this, dirt);
```

Comment:
This shouldn't be renamed. The name is from the game itself, and as "SDK" logic, we have to stick to it. You can declare a custom `SetVehicleDirty` method that calls the `PosefujZimuVShopu` as an alias.

Reply:
![image](https://github.com/MafiaHub/MafiaMP/assets/72361937/bfae9abe-d1d5-4f6f-a0e7-bdc6acf3e6cb)

 From M3, same code, pretty sure PosefujZimuVShopu has been removed in favor of SetVehicleDirty